packages/eslint-plugin/src/rules/jsx-pragma/index.ts

```diff
@@ -121,66 +166,38 @@ export const jsxPragmaRule: Rule.RuleModule = {
         }
       },
 
+      'JSXOpeningElement[name.name=/^[a-z]+$/] > JSXAttribute[name.name=/^className$/]': (
+        node: Rule.Node
+      ) => {
+        if (node.type !== 'JSXAttribute' || jsxPragma || jsxImportSourcePragma) {
+          return;
+        }
+
+        if (
+          node.value?.type === 'JSXExpressionContainer' &&
+          node.value.expression.type === 'Identifier' &&
+          /[Xx]css$/.exec(node.value.expression.name)
```

Comment:
using `test` instead of `exec` may be a bit faster

Reply:
Great, will update.